lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -151,7 +159,7 @@ class ElastiCacheManager(object):
     def __init__(self, module, name, engine, cache_engine_version, node_type,
                  num_nodes, cache_port, cache_parameter_group, cache_subnet_group,
                  cache_security_groups, security_group_ids, zone, wait,
-                 hard_modify, region, **aws_connect_kwargs):
+                 hard_modify, region, replication_group, **aws_connect_kwargs):
         self.module = module
         self.name = name
         self.engine = engine.lower()
```

Comment:
This fails now if replication_group is specified because engine and node_type are set to None in main and you can't .lower() None.